hugegraph-llm/requirements.txt

```diff
@@ -1,4 +1,4 @@
-openai==0.28.1
+openai
```

Comment:
It will be more reliable to specify a version(to avoid compatibility issues), such as >= a certain version